rlberry/manager/evaluation.py

```diff
@@ -151,6 +312,11 @@ def read_writer_data(data_source, tag=None, preprocess_func=None, id_agent=None)
         Note: the agent's save function must save its writer at the key `_writer`.
         This is the default for rlberry agents.
 
+    many_agent_by_str_datasource : bool (False by default)
+        (only useful if the datasource is a str or a list of str)
+        If True, select many agent : the last agent of every different agent name inside the folder
+        If False, select only one agent :the last trained agent from the directory (whatever its name)
```

Comment:
the agent that have been trained the latest? I am not really sure if there is a better way to say it but I think at least this is a bit better.